scene/2d/remote_transform_2d.cpp

```diff
@@ -63,7 +63,44 @@ void RemoteTransform2D::_update_remote() {
 		return;
 
 	//todo make faster
-	n->set_global_transform(get_global_transform());
+	if (use_global_coordinates) {
+
+		if (update_remote_position && update_remote_rotation && update_remote_scale) {
+			n->set_global_transform(get_global_transform());
+		} else {
+			Transform2D n_trans = n->get_global_transform();
+			Transform2D our_trans = get_global_transform();
+
+			if (!update_remote_position)
+				our_trans.set_origin(n_trans.get_origin());
+			if (!update_remote_rotation)
+				our_trans.set_rotation(n_trans.get_rotation());
+
+			n->set_global_transform(our_trans);
+
+			if (update_remote_scale)
+				n->set_global_scale(get_global_scale() + Vector2(1, 1));
```

Comment:
Setting transform sets scale too, and I couldn't find any way to change a transform's scale. So I set the scale after to get around it.